inc/upgrade.php

```diff
@@ -64,6 +64,24 @@ function responsive_framework_upgrade( $verbose = true ) {
 
 add_action( 'init', 'responsive_framework_upgrade' );
 
+/**
+ * Ensures that the default options are saved in the database.
+ */
+function responsive_upgrade_ensure_theme_options() {
+	// Ensure customizer options have default values saved.
+	$show_on_bottom = get_option( 'burf_setting_posts_sidebar_bottom' );
+
+	if ( empty( $show_on_bottom ) ) {
+		add_option( 'burf_setting_posts_sidebar_bottom', true );
```

Comment:
do we just want to set `burf_setting_posts_sidebar_bottom` if it isn't set already, or do we want to set it to true?

if the latter, we'd want to do something like the following:
```php
if( $show_on_bottom == false ){ update_option( 'burf_setting_posts_sidebar_bottom', true); }
```

Reply:
I think this current code is ok, cuz we only want to add the option if it doesn't exist yet during the upgrade function.